app/views/dashboard/_wiki.html.erb

```diff
@@ -14,7 +14,7 @@
   </div>
 
   <div class="col-xl-4 col-lg-5 col-md-5 col-5">
-    <a href="/wiki/new" class="btn btn-primary btn-sm btn-outline-secondary btn-block" type="button" style="color:white" ><i class="fa fa-plus-circle" style="color:white" ></i> <%= translation('dashboard._wiki.new_page') %></a>
+    <a href="/wiki/new" class="btn btn-primary btn-sm btn-outline-secondary btn-block" type="button" style="color:white" ><i class="fa fa-plus-circle" style="color:white; margin-left:-3px" ></i> <%= translation('dashboard._wiki.new_page') %></a>
```

Comment:
Hey @BenUnyolo , 
Thanks for opening the PR 😄 
We generally avoid using hardcoded numbers in css but I see the FTO issue had this solution given. 
Do you want to try using [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)  to wrap the `input` element and this button? 
or maybe find solution using bootstrap?
Let us know if you wish to merge this PR and take this up in a separate PR, no worries 😄 
Thanks 🎉

Reply:
Hey @Sagarpreet, makes sense! Thanks for the ideas, with me having difficulty getting the app running locally I wouldn't want to try and make changes without being able to see them so you can either ignore / reject this PR or merge and create a new one, I don't mind.